Description: Remove erroneous include path
Author: Fabian Wolff <fabi.wolff@arcor.de>
Forwarded: no
Last-Update: 2017-03-17
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/unit/Makefile.am
+++ b/test/unit/Makefile.am
@@ -69,7 +69,6 @@
 
 AM_CPPFLAGS = \
 	-I. \
-	"-I@CXXTEST@" \
 	"-I@top_builddir@/src" \
 	"-I@top_srcdir@/src/include" \
 	"-I@top_srcdir@/lib" \
